test/cipher_test.rb

```diff
@@ -1,7 +1,6 @@
 require_relative 'test_helper'
 require './lib/cipher'
 
-
 class CipherTest < Minitest::Test
   def setup
     Date.stubs(:today).returns(Date.new(1995, 8, 4))
```

Comment:
I'm not sure if best practice is to leave this date stub in the setup, or just have it in its test. 🤷‍♀️